__test__/op-connect.test.ts

```diff
@@ -458,6 +458,49 @@ describe("Test OnePasswordConnect CRUD", () => {
             expect(otp).toEqual(OTP);
         });
     });
+
+    describe("List items by title search", () => {
+        const title = "some title";
+        // Search all items by title which contains  the given value `title`
+        const getItemsByTitleMock = (title: string) => nock(mockServerUrl)
```

Comment:
You can use the `apiMock.listItemsByTitleSearch` method that you've created instead.